easy_my_coop/models/operation_request.py

```diff
@@ -282,23 +282,25 @@ def validate(self):
                     )
                 )
 
-    def get_share_trans_mail_template(self):
+    def _get_share_transfert_mail_template(self):
```

Comment:
same remarks about `transfert` and `…_mail_…` vs `…_email_…` as in coopiteasy/investor-wallet-platform#72.